scripts/flash/designated-report.js

```diff
@@ -9,13 +9,25 @@ var setTimestamp = require("./set-timestamp");
 var getRepTokens = require("./get-rep-tokens");
 var doInitialReport = require("./do-initial-report");
 var getPayoutNumerators = require("./get-payout-numerators");
-var getPrivateKeyFromString = require("../dp/lib/get-private-key").getPrivateKeyFromString;
+
+function help() {
+  console.log(chalk.red("This command will move time to the designate report time fram and do an initial-report"));
+  console.log(chalk.red("FYI: Scalar markets outcome is the value between min and max, use market-info for details"));
+}
 
 /**
  * Move time to Market end time and do initial report
  */
-function designateReportInternal(augur, marketId, outcome, userAuth, invalid, auth, callback) {
+function designateReport(augur, args, auth, callback) {
+  if (args === "help" || args.opt.help) {
+    help();
+    return callback(null);
+  }
   var amount = 10000;
+  var marketId = args.opt.marketId;
+  var userAuth = auth;
```

Comment:
It seems odd to set `userAuth` = `auth` here when later in the code there's this line:

```
if (userAuth.address !== market.designatedReporter && auth.address === market.designatedReporter) {
        userAuth = auth;
}
```